internal/logger/logger.go

```diff
@@ -153,41 +163,32 @@ type loggerFactory struct {
 	sysWriter *syslog.Writer
 	flag      int
 	format    string
+	level     config.LogSeverity
 }
 
-func (f *loggerFactory) newLogger(level, prefix string) *log.Logger {
-	return log.New(f.writer(level), prefix, f.flag)
+func (f *loggerFactory) newLogger(level config.LogSeverity) *slog.Logger {
+	// create a new logger
+	var programLevel = new(slog.LevelVar)
+	logger := slog.New(f.handler(programLevel, ""))
```

Comment:
Why are we not passing prefix?

Reply:
the default loggers didn't have any prefixes earlier so I kept the same behaviour here.